src/OSPSuite.Core/Domain/Services/ParameterIdentifications/ParameterIdentificationEngine.cs

```diff
@@ -102,6 +103,8 @@ private ParallelOptions createParallelOptions(CancellationToken token)
       public void Stop()
       {
          _cancellationTokenSource.Cancel();
+         _parameterIdentificationRuns.Each(x => x.Cancel());
+         _parameterIdentificationRuns.Clear();
```

Comment:
should that be done also on line 51 or on dispose? Otherwise it's never cleared unless we stop

Reply:
in the finally you mean right? line 71?